exporter/newrelicexporter/newrelic_test.go

```diff
@@ -213,9 +213,10 @@ func TestExportTraceDataMinimum(t *testing.T) {
 			ID:      "0000000000000001",
 			TraceID: "01010101010101010101010101010101",
 			Attributes: map[string]interface{}{
-				"collector.name":    name,
-				"collector.version": version,
-				"name":              "root",
+				"collector.name":               name,
+				"collector.version":            version,
+				"name":                         "root",
+				instrumentationProviderAttrKey: "opentelemetry",
```

Comment:
I would stick with string to make the test easier to read without having to double check against the code. Otherwise, let's update the other keys to use the variables - mixing strings and variables looks a bit odd.

Reply:
Good point, I'll revert the key to string.